### PR TITLE
Fix retrieving tags on place pages.

### PR DESCRIPTION
**Pull request by vlvereta**

### CODE REVIEW

File: app/Actions/Place/GetPlaceCollection/GetPlaceCollectionPresenter.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public function present(GetPlaceCollectionResponse $placeResponse): array
             $result['features'] = $this->featurePresenter->presentCollection($place->features);
             $result['localization'] = $this->localizationPresenter->presentCollection($place->localization);
             $result['category'] = $this->categoryPresenter->present($place->category);
-            $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->category->tags);
+            $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->tags);
```
Comment (nemoipaha) [inline app/Actions/Place/GetPlaceCollection/GetPlaceCollectionPresenter.php]: $place->tags loaded in repository with(...)?

File: app/Actions/Place/GetPlaceItem/GetPlaceItemPresenter.php [hunk 2]
```diff
@@ -61,7 +61,7 @@ public function present(GetPlaceItemResponse $placeResponse): array
         $result['features'] = $this->featurePresenter->presentCollection($place->features);
         $result['localization'] = $this->localizationPresenter->presentCollection($place->localization);
         $result['category'] = $this->categoryPresenter->present($place->category);
-        $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->category->tags);
+        $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->tags);
```
Comment (nemoipaha) [inline app/Actions/Place/GetPlaceItem/GetPlaceItemPresenter.php]: [place][tags]

Comment (vlvereta) [reply app/Actions/Place/GetPlaceItem/GetPlaceItemPresenter.php]: Corrected there and deeper also (in UserLists pages).

File: app/Actions/Place/GetPlaceCollection/GetPlaceCollectionPresenter.php (hunk 1 above)
Comment (vlvereta) [reply app/Actions/Place/GetPlaceCollection/GetPlaceCollectionPresenter.php]: Done.

**Pull request merged**